pkg/runtime/wfengine/workflow.go

```diff
@@ -498,6 +521,9 @@ func (wf *workflowActor) runWorkflow(ctx context.Context, actorID string, remind
 		}
 	}
 
+	// record time taken to complete the workflow
+	elapsed := diag.ElapsedSince(start)
+	diag.DefaultWorkflowMonitoring.ExecutionEvent(ctx, "dapr", diag.Workflow, diag.StatusSuccess, elapsed)
```

Comment:
@prateek041 This is actually not the correct time for workflow completion. The time calculated here just measures the time for One `Schedule` of workflow. There would be different schedules and the workflow completion time should be the sum of all that. As mentioned here: https://github.com/dapr/dapr/issues/7109 It requires some runtime changes. So for now we can just pass `0` in elapsed time to not record and add a comment that workflow execution latency is not supported yet.

Reply:
sure, fixed